File: CodingBat/Python/Warmup_1/pos_neg.py
#Given 2 int values return True if one is negative and one is positive. Except if the parameter "negative" is True, then return True only if both are negative.
def pos_neg(a, b, negative):
    if negative == True:
        if abs(a) != a and abs(b) !=b :
            return True
        return False
    else:
        if abs(a) != a and abs(b) != b:
            return False
        elif abs(a) == a and abs(b) == b:
            return False
        return True

File: CodingBat/Java/Warmup_1/posNeg.java
public class posNeg{
    // Given 2 int values, return true if one is negative and one is positive. Except if the parameter "negative " is true, then return true if both are negative.
    public static boolean posNeg(int a, int b, boolean negative){
        if (negative){
            if ((a<0)&&(b<0)){
                return true;
            }
        }
        else if(((a<0)&&(b>0))||((a>0)&&(b<0))){
            return true;
        }
        return false;
    }
    public static void main(String[] args) {
        
    }
}